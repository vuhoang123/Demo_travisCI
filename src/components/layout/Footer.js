import React from 'react'
import CssBaseline from '@material-ui/core/CssBaseline';

function Footer() {
  return (
    <React.Fragment>
      <CssBaseline />
      <footer className="footerStyle">
        <span>filmSearch â“’ 2022 built with</span>
        <img className="tmdbStyle" src="https://www.themoviedb.org/assets/2/v4/logos/408x161-powered-by-rectangle-green-bb4301c10ddc749b4e79463811a68afebeae66ef43d17bcfd8ff0e60ded7ce99.png" alt="" />
      </footer>
    </React.Fragment>
  )
}

export default Footer;
