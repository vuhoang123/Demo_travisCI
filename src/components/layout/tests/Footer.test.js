import React from 'react';
import Footer from '../Footer';
import renderer from 'react-test-renderer';

describe('Footer', () => {
  
  it('matches snapshot', () => {
    
    const tree = renderer.create(
      <Footer/>
    ).toJSON();
    
    expect(tree).toMatchSnapshot();
  });
  
});
