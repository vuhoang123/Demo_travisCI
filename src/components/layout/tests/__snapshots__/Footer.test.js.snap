// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer matches snapshot 1`] = `
<footer
  className="footerStyle"
>
  <span>
    filmSearch â“’ 2019 built with
  </span>
  <img
    alt=""
    className="tmdbStyle"
    src="https://www.themoviedb.org/assets/2/v4/logos/408x161-powered-by-rectangle-green-bb4301c10ddc749b4e79463811a68afebeae66ef43d17bcfd8ff0e60ded7ce99.png"
  />
</footer>
`;
